CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ it can be used for such and such.
 To deliver this feature we had to extend, this that and the other
 with new behaviours
 
-Backwards compatability is kept
+Backwards compatibility is kept
```

Comment:
hah, I am going to have to fix so many of these :) every project has this mistake lol